### ISSUE TITLE
Polyp Segmentation in Endoscopy Image

### ISSUE THREAD

**Issue by Kaushal-11** [SrijanShovit/HealthLearning#124]
### Is your feature request related to a problem? Please describe.

I propose to add a feature for Polyp Segmentation using an endoscopy image dataset. This feature aims to improve the detection and segmentation of polyps in endoscopic images, which is crucial for early diagnosis and treatment of colorectal cancer. 

Kindly assign me the issue under the label of GSSoC'24.

### Describe the solution you'd like

Dataset Link : https://polyp.grand-challenge.org/CVCClinicDB/

Model :  U - Net model is used for segmentation with using tensorflow library.

Data Preprocessing: Using Pandas and NumPy for cleaning, normalizing, and preparing the dataset.
Model Development: Leveraging TensorFlow and Keras to build and train a deep learning model for polyp segmentation.
Evaluation: Implementing techniques to evaluate model performance, such as IoU (Intersection over Union) and Dice coefficient, to ensure high accuracy and precision.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Kaushal-11** [comment]
@SrijanShovit , Can i start to work on this feature??

**Comment by Kaushal-11** [comment]
i want to work on this issue,. Please assign to me under Gssoc'24

**Comment by SrijanShovit** [comment]
Yes @Kaushal-11  proceed

**Comment by SrijanShovit** [comment]
https://github.com/SrijanShovit/HealthLearning/pull/56 go through this first

**Comment by SrijanShovit** [comment]
any updates @Kaushal-11

**Comment by Kaushal-11** [comment]
Yes, the task is almost complete. 
The model's training accuracy has reached 0.96, and the validation accuracy is at 0.93.

**Comment by Kaushal-11** [comment]
why is my issue is closed?
I am working on this issue..

**Comment by SrijanShovit** [comment]
Automation stuff. Just make a documentation of your approach as well.

**Comment by Kaushal-11** [comment]
Could you please provide me with a reference or guide on how to document my process?

**Comment by SrijanShovit** [comment]
You can check the codes of @Arihant-Bhandari under Metabolic Syndrome, @aditi1807 under Cirrhosis and @theiturhs  under MRI Classification

**Comment by Kaushal-11** [comment]
Okay, got it

**Comment by Kaushal-11** [comment]
I created pull request yesterday,
can you check it?

**Comment by Kaushal-11** [comment]
Hello @SrijanShovit ,
can you check my stuff?
